### PR TITLE
Admin only: Pair pinpointers, come in matched pairs, and always point to the other one

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Admin and event only pair pinpointers! They come in a box of two, and each pinpointer will always point at its corresponding pair. Aww.
/:cl:

Would come in handy in tracking your date for the Winter Ball.

The key item path here is `/obj/item/storage/box/pinpointer_pairs`, which makes a box containing
two pinpointers that are paired with each other.

### CODE REVIEW

**Comment by CitrusGender** [comment]
oml so cute

**Comment by optimumtact** [comment]
it should still have a changelog

**Comment by GhostEcho** [comment]
Im concerned with how much this will be abused by metafriends and shit like that

**Comment by GhostEcho** [comment]
oh wait Im dumb

**Comment by subject217** [comment]
what part of
>it's only admin/event accessible

do you not understand ghostecho

**Comment by GhostEcho** [comment]
I JUST READ THE DESCRIP AFTER SEEING THE TITLE POP UP IN GAME REEE

**Comment by GhostEcho** [comment]
I can appreciate this otherwise with it being admin/event only

**Comment by Qustinnus** [comment]
>tfw lose ball date
>use my girlfriend tracking device to find her
>walk into dorms as it points to where my beloved Helen Weinstein is
>open Dorm room 3
>then i see it, chad thundercock standing over my beloved helen weinstein


smh ball ruined

**Comment by anconfuzedrock** [comment]
suit sensors

**Comment by Anonmare** [comment]
@Qustinnus Your date is mine yet again nerd

File: code/game/objects/items/pinpointer.dm [hunk 1]
```diff
@@ -140,3 +141,35 @@
...
+	name = "pair pinpointer"
+	desc = "A handheld tracking device that locks onto its other half of the matching pair."
+	var/other_pair
+
+/obj/item/pinpointer/pair/Destroy()
+	other_pair = null
+	. = ..()
+
+/obj/item/pinpointer/pair/scan_for_target()
+	target = other_pair
```
Comment (AlexTheSysop42) [inline code/game/objects/items/pinpointer.dm]:
```suggestion
	target = other_pair
	. = ..()
```

Comment (coiax) [reply code/game/objects/items/pinpointer.dm]: The parent method does nothing.

**Pull request merged**